.eslintrc.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
   },
   plugins: ['@typescript-eslint'],
   rules: {
+    'import/named': 'off',
```

Comment:
Will want to give this eslint config a good look. Right now there's mixed `module.export` and `export default` leading to warnings that would be resolved if all `module.exports` were refactored out. However, there might be a setting allowing for better interop.